# 7.2 L, L over D and K from Kv for Conventional Type Valves

A 150 mm class 125 Y-pattern globe valve with a Kv=500 flow coefficient is given.

Calculate the resistance coefficient K, L/D equivalent, and the length for complete turbulence in the flow.

Use schedule 40, 150 mm pipe as a reference.

In [1]:
from fluids.units import *
D = .154*u.m
Kv = 500*u.m**3/u.hour

K = Kv_to_K(Kv, D)
L_D = L_equiv_from_K(K, fd=.015)
L = D*L_D

print('Loss coefficient = %s' %K)
print('Equivalent length = %s' % L_D)
print('Length for complete turbulence = %s' %L)

Loss coefficient = 3.5996713983999995 dimensionless
Equivalent length = 239.97809322666663 dimensionless
Length for complete turbulence = 36.95662635690666 meter


The results calculated in Crane TP 410M are  3.6, 240, and 37.0 respectively.